Lib/fontTools/ttLib/tables/G__l_o_c.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import print_function, division, absolute_import
+from fontTools.misc.py23 import *
+from fontTools.misc import sstruct
+from fontTools.misc.textTools import safeEval
+from itertools import *
+from functools import partial
+from . import DefaultTable
+import array
+import struct
+import operator
+import warnings
+from _ast import Num
```

Comment:
this import seems unused

Reply:
Cleared out the redundant imports there. I'll use graphiteUtils if you use opentypeUtils :) gr || ot just as graphite || opentype.